.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,35 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- See [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
+*  KFP version: <!-- See version number shows on bottom of KFP UI left sidenav. -->
```

Comment:
Suggested revision, please update all instances:

Specify the version of Kubeflow Pipelines that you are using. The version number appears in the user interface; in the left side navigation.
To find the version number, See version number shows on bottom of KFP UI left sidenav.

Reply:
Sounds good! Done with a small adjustment